dcp_prototype/backend/wrangling/migrations/metadata_schema_representation/old_entities/old_dataset_metadata.py

```diff
@@ -42,6 +42,19 @@
     hca_accession_generator,
     hca_accession_transformer,
 )
+import threading
+
+ENTITY_TYPES = [
```

Comment:
would it make sense to move this to https://github.com/HumanCellAtlas/dcp-prototype/blob/99402901041f4bb05ff4190e417dbe45d836d598/dcp_prototype/backend/wrangling/migrations/utils/constants.py?

Reply:
Yep, that's a reasonable suggestion.

Reply:
okay, I'll move that